rust/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasm-xlsxwriter"
-version = "0.10.0"
+version = "0.11.0-alpha.1"
```

Comment:
We have to release with `npm publish --provenance --access public --tag alpha` to make sure it is released as alpha release.